meson.build

```diff
@@ -74,18 +75,38 @@ if get_option('wayland') != 'false'
   endif
 endif
 
+# WIN32
+use_win32 = false
+if get_option('win32') != 'false' and host_machine.system() == 'windows'
```

Comment:
Mid/long term question:

Ideally we'd want va-win32 detection and building to work with mingw on top of Linux. Both here and in libva itself.

Reply:
Sounds good. Enabling usage of win32 should only depend on libva-win32 being present, disregarding the platform